### PR TITLE
Cloud Docker release 1.2.1

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

This pull request (PR) updates the release notes and related documentation for Cloud Docker 1.2.1.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html
- https://devdocs.magento.com/cloud/docker/docker-containers-service.html
- https://devdocs.magento.com/cloud/docker/docker-containers.html
- https://devdocs.magento.com/cloud/docker/docker-quick-reference.html

whatsnew

- [Cloud Docker 1.2.1 release notes](https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html).
- New Cloud Docker build command option for generating a Docker configuration without a [TLS container](https://devdocs.magento.com/cloud/docker/docker-containers-service.html#tls-container).
- New Cloud Docker build commands for configuring NGINX for the [Web container](https://devdocs.magento.com/cloud/docker/docker-containers-service.html#web-container).

### CODE REVIEW

File: src/cloud/docker/docker-containers-service.md [hunk 1]
```diff
@@ -267,6 +269,11 @@ docker-compose exec varnish varnishadm ban req.url '~' '.'
 
 The Web container uses NGINX to handle web requests after TLS and Varnish. This container passes all requests to the FPM container to serve the PHP code. See [Request flow]({{site.baseurl}}/cloud/docker/docker-containers.html#request-flow).
 
+This container provides two NGINX configuration options:
+
+-  `--nginx-worker-processes` to set the number of worker processes for NGINX. Default is `1`.
+-  `--nginx-worker-connections` to set maximum number of connections that each worker process can handle simultaneously. The default is set to `1024`.
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:275]:
```suggestion
-  `--nginx-worker-connections` to set the maximum number of connections that each worker process can handle simultaneously. The default is `1024`.
```

File: src/cloud/docker/docker-containers-service.md [hunk 2]
```diff
@@ -267,6 +269,11 @@ docker-compose exec varnish varnishadm ban req.url '~' '.'
 
 The Web container uses NGINX to handle web requests after TLS and Varnish. This container passes all requests to the FPM container to serve the PHP code. See [Request flow]({{site.baseurl}}/cloud/docker/docker-containers.html#request-flow).
 
+This container provides two NGINX configuration options:
+
+-  `--nginx-worker-processes` to set the number of worker processes for NGINX. Default is `1`.
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:274]:
```suggestion
-  `--nginx-worker-processes` to set the number of worker processes for NGINX. The default is `1`.
```

File: src/cloud/release-notes/mcd-release-notes.md [hunk 3]
```diff
@@ -16,6 +16,30 @@ The release notes include:
...
+*Release date: December 21, 2020*<br/>
+
+-  {:.new}<!--MCLOUD-7259-->**NGINX command options.** Added build command options to change the number of NGINX `worker_processes` and NGINX `worker_connections` for TLS and Web services. The `worker_process` parameter retains the ability to set the value to `auto`. Examples:
+
+    ```terminal
+    ./vendor/bin/ece-docker build:compose --nginx-worker-processes=2
+    ./vendor/bin/ece-docker build:compose --nginx-worker-connections=2048
+    ```
+
+-  {:.new}<!--MCLOUD-7259-->**TLS command option.** Added build command option to create a configuration without the TLS service. Example:
```
Comment (meker12) [inline src/cloud/release-notes/mcd-release-notes.md:29]:
```suggestion
-  {:.new}<!--MCLOUD-7259-->**TLS command option**–Added build command option to create a configuration without the TLS service. Example:
```

File: src/cloud/release-notes/mcd-release-notes.md [hunk 4]
```diff
@@ -16,6 +16,30 @@ The release notes include:
...
+   ```
+
+-  {:.new}<!--MCLOUD-7259-->**NGINX memory consumption**. Reduced the memory consumed by the NGINX process for TLS and Web services.
+
+-  {:.new}<!--No ticket -->**Blackfire**. Disabled Blackfire PHP extension by default in the Cloud Docker image.
+
+-  {:.fix}<!--MCLOUD-7232-->**PHP-FPM container**. Fixed PHP-FPM container health check by changing the `WEB_PORT` from `80` to `8080`.
+
+-  {:.fix}<!--MCLOUD-7442-->**Invalid volume naming**. Fixed an error with invalid volume naming in developer mode.
+
```
Comment (meker12) [inline src/cloud/release-notes/mcd-release-notes.md:42]:
```suggestion
-  {:.new}<!--MCLOUD-7259-->**NGINX memory consumption**–Reduced the memory consumed by the NGINX process for TLS and Web services.

-  {:.new}<!--No ticket -->**Blackfire**–Disabled Blackfire PHP extension by default in the Cloud Docker image.

-  {:.fix}<!--MCLOUD-7232-->**PHP-FPM container**–Fixed PHP-FPM container health check by changing the `WEB_PORT` from `80` to `8080`.

-  {:.fix}<!--MCLOUD-7442-->**Invalid volume naming**–Fixed an error with invalid volume naming in developer mode.

```

File: src/cloud/release-notes/mcd-release-notes.md [hunk 5]
```diff
@@ -16,6 +16,30 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v1.2.1
+*Release date: December 21, 2020*<br/>
+
+-  {:.new}<!--MCLOUD-7259-->**NGINX command options.** Added build command options to change the number of NGINX `worker_processes` and NGINX `worker_connections` for TLS and Web services. The `worker_process` parameter retains the ability to set the value to `auto`. Examples:
```
Comment (meker12) [inline src/cloud/release-notes/mcd-release-notes.md:22]:
```suggestion
-  {:.new}<!--MCLOUD-7259-->**NGINX command options**–Added build command options to change the number of NGINX `worker_processes` and NGINX `worker_connections` for TLS and Web services. The `worker_process` parameter retains the ability to set the value to `auto`. Examples:
```

File: src/cloud/docker/docker-containers-service.md [hunk 6]
```diff
@@ -267,6 +269,11 @@ docker-compose exec varnish varnishadm ban req.url '~' '.'
 
 The Web container uses NGINX to handle web requests after TLS and Varnish. This container passes all requests to the FPM container to serve the PHP code. See [Request flow]({{site.baseurl}}/cloud/docker/docker-containers.html#request-flow).
 
+This container provides two NGINX configuration options:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:272]:
```suggestion
This container provides two NGINX configuration options for building the Docker configuration:
```

**Review by meker12 — APPROVED** [review]

**Comment by bdenham** [comment]
running tests

**Comment by jeff-matthews** [comment]
@bdenham, for future release PRs, please don't forget to add a Major update label and whatsnew entry to the Pr description.

**Comment by bdenham** [comment]
Yep, I remembered the whatsnew stuff, just forgot to tag it with the major update label.

**Pull request merged**